app/frontend/src/features/messages/messagelist/MessageView.tsx

```diff
@@ -1,55 +1,123 @@
-import { Box, Card, CardContent, IconButton } from "@material-ui/core";
+import { Box, Card, CardContent, Typography } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import ExpandLessIcon from "@material-ui/icons/ExpandLess";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
-import React, { useState } from "react";
+import React from "react";
 import Avatar from "../../../components/Avatar";
 import { Message } from "../../../pb/conversations_pb";
 import { timestamp2Date } from "../../../utils/date";
 import useCurrentUser from "../../userQueries/useCurrentUser";
 import { useUser } from "../../userQueries/useUsers";
 import TimeInterval from "./MomentIndication";
-import UserName from "./UserName";
+import classnames from "classnames";
 
-const useStyles = makeStyles({
-  root: { display: "flex" },
-  card: { flexGrow: 1 },
-  header: { display: "flex" },
-  name: { flexGrow: 1 },
-});
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    marginTop: theme.spacing(2),
+    "& > :first-child": { marginRight: theme.spacing(2) },
+  },
+  card: {
+    [theme.breakpoints.up("xs")]: {
+      width: "100%",
+    },
+    [theme.breakpoints.up("sm")]: {
+      width: "80%",
+    },
+    [theme.breakpoints.up("md")]: {
+      width: "70%",
+    },
+    border: "1px solid",
+    borderRadius: theme.shape.borderRadius,
+  },
+  userCard: {
+    borderColor: theme.palette.secondary.main,
+  },
+  otherCard: {
+    borderColor: theme.palette.primary.main,
+  },
+  header: {
+    display: "flex",
+    padding: theme.spacing(2),
+    paddingBottom: theme.spacing(1),
+    alignItems: "center",
+  },
+  footer: {
+    display: "flex",
+    paddingInline: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+    justifyContent: "flex-end",
+  },
+  messageTime: {
+    color: theme.typography.caption.color,
+    fontSize: theme.typography.caption.fontSize,
+    paddingInlineEnd: theme.spacing(1),
+  },
+  avatar: { width: 40, height: 40 },
+  name: {
+    flexGrow: 1,
+    margin: 0,
+    fontSize: theme.typography.body2.fontSize,
+    fontWeight: "bold",
+  },
+  messageBody: {
+    paddingTop: 0,
+    paddingBottom: theme.spacing(1),
+    "&:last-child": { paddingBottom: theme.spacing(2) },
+  },
+}));
 
 export interface MessageProps {
   message: Message.AsObject;
 }
 
 export default function MessageView({ message }: MessageProps) {
-  const [expanded, setExpanded] = useState<boolean>(false);
   const classes = useStyles();
   const { data: author } = useUser(message.authorUserId);
   const { data: currentUser } = useCurrentUser();
   const isCurrentUser = author?.userId === currentUser?.userId;
   return (
-    <Card className={classes.root}>
-      {author && !isCurrentUser && <Avatar user={author} />}
-      <Box className={classes.card}>
+    <Box
+      className={classes.root}
+      style={{ justifyContent: !isCurrentUser ? "flex-start" : "flex-end" }}
```

Comment:
Couldn't we do something similar to what you did on line 86 here and conditionally apply class names so all styles are in the `makeStyles(...)` for consistency?